src/test/resources/pages/instructorFeedbackResultsSortQuestion.html

```diff
@@ -2710,114 +2710,6 @@
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                        <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
-                                                              <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
```

Comment:
I did an investigation and managed to find out why. This is not a bug, but rather an unfortunate series of event. To explain, let's get back to about three weeks ago.
I made this commit: ac68b6c, which basically is running God mode through one UI test and getting one whole bunch of test files updated with that comment line. Up to here it still looks fine.
Then this commit is made: ba08c70. Looking at the issue being addressed, I understand that a large amount of line is removed.
The sum of addition of the comment lines and removal of those existing lines is what we see up there.